acdc-ws/app/controllers/MetricController.scala

```diff
@@ -0,0 +1,23 @@
+package controllers
+
+import play.api.mvc._
+import services.Metric
+
+import javax.inject._
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class MetricController @Inject() (
+  cc: ControllerComponents,
+  metric: Metric
+ ) (implicit
+    ec: ExecutionContext
+ )  extends AbstractController(cc) {
+
+  def collect: Action[AnyContent] = Action.async { r: Request[AnyContent] =>
+      val metricResults = metric.collect.map(output => Ok(output))
+      metricResults.onComplete(_ => metric.clear())
+      metricResults
+    }
```

Comment:
can you format with the supplied `.scalafmt.conf`?

Reply:
Yes, I ran the project via the supplied .scalafmt.conf.